cmd/image-factory/cmd/service.go

```diff
@@ -183,9 +183,14 @@ func buildAssetBuilder(logger *zap.Logger, artifactsManager *artifacts.Manager,
 
 	builderOptions.RemoteOptions = append(builderOptions.RemoteOptions, remoteOptions()...)
 
+	repoOpts := []name.Option{}
```

Comment:
Linter should complain about that no?

Should be
`var repoOpts []name.Option`

Reply:
I don't think it really matters here, but I will fix it